src/views/InternalApp/TheBets/TheBets.vue

```diff
@@ -30,7 +30,18 @@ function selectBet(event: string) {
   betStore.selectedBet = betStore.bets?.find((bet) => bet._id === event)!;
 }
 
+function changePage(newPage: number) {
+  loadBets(newPage)
+}
+
+async function loadBets(page: number) {
+  await betStore.getBets(page, 10);
+}
+
+
 onMounted(async () => {
+  await betStore.getBets(betStore.currentPage, 10); 
```

Comment:
if we are fetching here, why are we fetching data again without any param below, please remove one of them with the proper query